Tests/APITesters/ObjCAPITester/ObjCAPITester.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		2C396F5E281C64B700669657 /* AdServices.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2C396F5D281C64B700669657 /* AdServices.framework */; };
```

Comment:
⚠️ I think we shouldn't do this. The fact that `APITester`s needs it is a sign that users will need to add it as well, which isn't good. I'm guessing the actual problem was this: #1554.

Reply:
agreed. promise i won't forget this before merging to main.